L1Trigger/L1THGCal/test/HGCalTriggerBestChoiceTester.cc

```diff
@@ -48,27 +58,46 @@ class HGCalTriggerBestChoiceTester : public edm::EDAnalyzer
     private:
         void checkSelectedCells(const edm::Event&, const edm::EventSetup&);
         void rerunBestChoiceFragments(const edm::Event&, const edm::EventSetup&);
-        void fillModule(const std::vector<HGCDataFrame<HGCalDetId,HGCSample>>&,
-                const HGCalBestChoiceDataPayload&,
-                const vector<pair<HGCalDetId, uint32_t > >& );
+        void fillModule(const std::vector<HGCDataFrame<HGCalDetId,HGCSample>>&, const vector<pair<HGCalDetId, uint32_t > >&, const HGCalBestChoiceDataPayload&,  const HGCalBestChoiceDataPayload&,const HGCalBestChoiceDataPayload&,   map <HGCalDetId,double>&,  map<uint32_t, double>& );
```

Comment:
Use const for the two last parameters since they are not modified by the function